RecoTracker/ConversionSeedGenerators/python/PhotonConversionTrajectorySeedProducerFromSingleLeg_cfi.py

```diff
@@ -23,7 +23,7 @@
                                                   ClusterCheckPSet = cms.PSet(
                                                       PixelClusterCollectionLabel = cms.InputTag("siPixelClusters"),
                                                       MaxNumberOfCosmicClusters = cms.uint32(400000),
-                                                      doClusterCheck = cms.bool(True),
+                                                      doClusterCheck = cms.bool(False),
```

Comment:
Nothing in iterative tracking uses (or should use) that one anymore, but fine for me for now.